CHANGELOG.md

```diff
@@ -7,18 +7,131 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+  participating in the consensus [#58]
+- Change `stake` and `withdraw` cryptographic signatures to what the stake
+  contract expects in the new spec [#58]
+- Change `StakeInfo` to have an optional amount staked, a reward, and a
+  signature counter [#58]
+- Change signature of `fetch_notes` by removing block height
+- Delegate buffer allocation in FFI `fetch_notes` to the user. [#58]
+- Note picker now picks the largest amount of notes possible, with the minimum
+  total value [#55]
+- Made `get_balance` return total and max spendable [#53]
+- Return early from `fetch_existing_nullifiers` in FFI [#49]
+- Modify `StateClient` to receive full stake info [#46]
+- Change `Canon` encoding length of `Transaction` [#31]
+- Return `Transaction` from transacting functions [#40]
+- Update transaction to generate change output
+- Crossover is now optional in transactions
+- Update STCT to use Schnorr signatures [#34]
+- Update dependencies
+
+### Fixed
+
+- Fix `encoded_len` in `Transaction` [#44]
+- Fix notes from `fetch_notes` being assumed unspent [#41]
+- Fix fee generation in stake and withdrawal
+
+### Removed
+
+- Remove `get_block_height` from the `StateClient` trait [#58]
+- Delete `extend_stake` since the stake contract removed it [#46]
+
+## [0.5.1] - 2021-01-26
+
+### Fixed
+
+- Stack overflow by allocating `fetch_notes` buffer on the heap [#25]
+
+## [0.5.0] - 2021-01-25
+
+### Changed
+
+- Stake operations now use BLS keys instead of JubJub keys [#22]
+
+## [0.4.0] - 2021-01-24
+
+### Changed
+
+- Update `rusk-abi` [#20]
+
+## [0.3.0] - 2021-01-23
 
 ### Added
 
-- Serialization and deserialization of transactions.
-- Temporary deterministic wallet generation.
-- Implementation of `get_balance` and `public_spend_key`.
-- Preliminary implementation of `create_transfer_tx`.
-- Expose `NodeClient` and `Store` through FFI.
-- Define FFI and compile it only for WASM.
-- `NodeClient` and `Store` traits encapsulating host dependant functionality.
-- Initial commit.
+- Implemented `Canon` for `Transaction` [#16]
+
+### Changed
+
+- Rename `to_bytes` to `to_var_bytes` when the struct is variable sized [#17]
+
+## [0.2.1] - 2021-01-22
+
+### Added
+
+- Publish built WASM to Github NPM [#1]
+- Add `malloc` and `free` to FFI [#13]
+
+## [0.2.0] - 2021-01-20
+
+### Added
+
+- Implemented `stake` and `withdraw` [#6]
+
+## [0.1.0] - 2021-01-13
+
+### Added
+
+- Serialization and deserialization of transactions
+- Temporary deterministic wallet generation
+- Implementation of `get_balance` and `public_spend_key`
+- Preliminary implementation of `create_transfer_tx`
+- Expose `NodeClient` and `Store` through FFI
+- Define FFI and compile it only for WASM
+
+<!--
+    `Added`: for new features
+    `Changed`: for changes in existing functionality
+    `Deprecated`: for soon-to-be removed features
+    `Removed`: for now removed features
+    `Fixed`: for any bug fixes
+    `Security`: in case of vulnerabilities
+-->
```

Comment:
This should be removed, there is already the link on the header that provides this info. Also, it doesn't looks like is related to anything come after (the issues).